infrastructure/stacks/after-lambda-deployment/alerts.tf

```diff
@@ -115,7 +115,7 @@ resource "aws_cloudwatch_metric_alarm" "change_request_to_dos_latency_alert" {
 }
 
 
-resource "aws_cloudwatch_metric_alarm" "change_request_sqs_dlq_alert" {
+resource "aws_cloudwatch_metric_alarm" "change_request_sqs_dql_alert" {
```

Comment:
please change the name from "change_request_sqs_dql_alert" to "change_request_sqs_dlq_alert"